packages/core/template_project/app/src/main/AndroidManifest.xml

```diff
@@ -105,11 +105,28 @@
             <meta-data android:name="android.support.customtabs.trusted.FALLBACK_STRATEGY"
                 android:value="@string/fallbackType" />
 
+            <% if (shareTarget) { %>
```

Comment:
Why not combine this section and the one you added below? Put both bits inside a single `if (shareTarget)`?